### ISSUE TITLE
Fejl i håndtering af geografiske koordinater i koordinatinput

### ISSUE THREAD

**Issue by kbevers** [SDFIdk/Koordinattransformation#87]
Når et koordinatsystem, hvor koordinaterne angives som geografiske koordinater, går kuk i input-felterne. Det er ikke muligt at skifte mellem de forskellige måder at repræsentere geografiske koordinater på (dd, min, min sek). Se eksempler herunder fra version 1.0.1: 
![billede](https://github.com/SDFIdk/Koordinattransformation/assets/13132571/2f65c6d2-d0bc-4a8a-88f5-1f7e72ced98a)
![billede](https://github.com/SDFIdk/Koordinattransformation/assets/13132571/334ec868-0a18-4bd9-b7c0-b82122cb6f7e)
![billede](https://github.com/SDFIdk/Koordinattransformation/assets/13132571/135a3a9e-5d56-45ae-a25c-ed0295e4bd72)

I seneste version (1.1.0) opdateres input-boksen ikke til visningen der gør det muligt at indtaste geografiske koordinater. Se herunder.

![billede](https://github.com/SDFIdk/Koordinattransformation/assets/13132571/4b127f2d-b260-4972-9c4d-b1fb01c96f9a)

**Comment by AAFredsted** [comment]
Ny udgave af siden er blevet skrevet, som findes dokumenteret i https://github.com/SDFIdk/Koordinattransformation/pull/95.

![Image](https://github.com/user-attachments/assets/bd7fee5a-4155-4948-a64a-e0e2a8bea646)

**Comment by kbevers** [comment]
Siden ser rigtig flot ud, @AAFredsted! Er den ved at være klar til at blive testet?

Mht D.min og D.min.sec repræsentation, så kunne det være fint også at få enheder på minutterne og sekunder. Breddegraden på billedet er med fuld enhedsangivelse 56°48'5.8776". Altså `'` for minutterne og `"` for sekunderne.

Teksten i formatvælgeren, de tre radiobuttons, er jeg heller ikke så vild med. Jeg undersøger lige om ikke der er en standardiseret måde at angive den slags på og vender tilbage med et ændringsønske hvis jeg finder noget der er mere tilfredstillende.

**Comment by kbevers** [comment]
Det ser godt ud, dog må der også gerne på input-siden komme enhedsangivelse på minutter (') og sekunder ("). Derudversynes jeg vi skal gå med følgende til radioknapperne:

```
DD.DD°
DD° MM.MM'
DD° MM' SS.SS"
```

Og det kunne også være fint hvis der indsættes et tooltip med teksten

> Repræsentationsformat for geografiske koordinater. Vælg mellem decimalgrader (DD.DD°), grader og minutter (DD° MM.MM') eller grader, minutter og sekunder (DD° MM' SS.SS").

Så tror jeg vi gjort det så grundigt som man kan.

**Comment by AAFredsted** [comment]
Tak for input :) det lyder som en god plan. 
Har fået rettet koordinatangivelsen til og lavet tooltips. 

![Image](https://github.com/user-attachments/assets/58777ad3-8c51-4d0f-9a59-1abcc00b4df3)

Kan du også give en opdateret udgave af teksten på https://koordinattransformation.dk/About? 

 @kbevers .

**Comment by AAFredsted** [comment]
@kbevers genskrivningen fra [Pull Request 95](https://github.com/SDFIdk/Koordinattransformation/pull/95) er deployed på https://test10.dataforsyningen.dk/ nu, så du må gerne tage et kig på den og dokumentere eventuelle mangler/fejl, som du støder på.

**Comment by kbevers** [comment]
Fedt. Det ser super godt ud. En enkelt ting har jeg opdaget lige med det samme: Når man trykker på kortet placeres "boblen" lidt nordøst for det egentlige musetryk. Jeg har forsøgt at lave en GIF:

![Image](https://github.com/user-attachments/assets/aaa4b829-2f6b-46d8-a742-c36aa29927a1)

Jeg vil forvente at boblen placeres lige midt i det kryds, der udgør musemarkøren.

Jeg beder mit kontor om at teste siden af. Jeg vender tilbage i næste uge med det de måtte finde.

**Comment by AAFredsted** [comment]
Det lyder godt @kbevers - Jeg har lavet en PR til at implementere ændringer baseret på den feedback, som der måtte komme. [Pull Request 97](https://github.com/SDFIdk/Koordinattransformation/pull/97)

**Comment by kbevers** [comment]
Her kommer første runde feedback:


### Navneord skrives ikke med stort begyndelsesbogstav

"Søg Koordinat via Adresse eller Stednavn" -> "Søg koordinat via adresse eller stednavn"

![Image](https://github.com/user-attachments/assets/227bba17-8c57-4596-a375-29295f82765c)

### Tooltips over koordinatkomponenter på inputsiden

Det vil være godt at have en hjælpetekst ved koordinatinputfelterne. Symbolerne før felterne antyder hvad der skal stå i hvert felt, men det kan godt være mere eksplicit. Et tooltip er vist nemmeste løsning uden at pille for meget ved designet. Jeg forestiller mig at teksten kommer frem når musen holdes over både symbol og selve feltet. Teksten i tooltippet kan trækkes fra WEBPROJ, hvor de fremgår som `v1`, `v2` og `v3` på CRS opslag. Et simpelt mockup herunder. 

![Image](https://github.com/user-attachments/assets/5fa3885e-7bd1-48b0-9663-7d74662bacf6)

### Cookie-erklæring?

Det er blevet påpeget at der ikke kommer en boks op og spørger om lov til at lagre cookies. Hvis der ikke er nogle cookies på siden er det jo i så fald fint, men tit har vi noget siteimprove eller lign. kørende som rigtige gerne vil gemme cookies. Så mest af alt er det her punkt bare et tjek for om der er orden i sagerne. 

### Tilgængelighedserklæring indeholder fejl

Det skal vist ikke løses her, men jeg nævner det da jeg ikke er klar over hvor det faktisk skal ændres.

![Image](https://github.com/user-attachments/assets/8d2571c7-d7a7-4db7-940b-18dc4eca3486)

### Decimalseparatorer er forskellige i input og output

I input bruges der komma, og i output bruges der punktum. Det skal være ens. Det mest rigtige, på dansk, er vist at bruge komma. I så fald vil en outputkoordinat da skulle formateres som `723910,4400 m 6179652,8900 m 123,0 m`. Alternativt skal input feltet bruge punktum som decimalseparator. Det er ikke så vigtigt for mig om det er den ene eller anden måde det gøres på.

### Afstand mellem tal og enhed på outputkoordinat

Der skal være mellemrum mellem tal og enhed, så `557314.0784m, 6202122.3108m, 0 m` i stedet bliver til `557314.0784 m, 6202122.3108 m, 0 m`. 

![Image](https://github.com/user-attachments/assets/84a76117-cff9-431c-8932-b6d088922fb0)

**Comment by AAFredsted** [comment]
Tak for input @kbevers  :) 
Cookies er noget som tilføjes eksternt når man deployer til produktion mener jeg, men jeg tager et kig på, 
om det kræver lidt ekstra opsætning fra min side.

**Comment by AAFredsted** [comment]
QA-pullrequest er blevet godkendt og er ude i test nu :) tag gerne et kig på den, når i finder tiden.

 Se [Pull Request 97](https://github.com/SDFIdk/Koordinattransformation/pull/97) for mere.

**Comment by AAFredsted** [comment]
@kbevers tak for at feedback :) Jeg får fixet outputkoordinaterne, og sendt et pull request her om lidt. I forhold til søgningen, ville jeg vente med den. Der er fokus på at få historiske kort til at overgå til arkivmeta v3 snarest, så jeg vil fokusere på at overholde non-funktionelle krav på koordinattransformation nu.

**Comment by kbevers** [comment]
> så jeg vil fokusere på at overholde non-funktionelle krav på koordinattransformation nu.

Helt fint. Jeg tænker også det vil være fint at få afsluttet opgaven her. Har du tjek på hvordan den rulles ud i produktion?

**Comment by AAFredsted** [comment]
@kbevers Den skal have et tag, og så deployer man tagget ikke?

Det kunne være rart med en hurtig gennemgang, hvis der en tid :)

**Comment by kbevers** [comment]
Jo, noget i den stil. Du må gerne indkalde mig til et møde efter påsken, så kan vi se på det sammen

**Comment by MAFredsted** [comment]
Issue løst i 1.0.2
